server/services/v1/search/session.go

```diff
@@ -97,8 +98,10 @@ func (sessions *SessionManager) TxExecute(ctx context.Context, runner TxRunner,
 		return Response{}, errors.NotFound("tenant '%s' not found", namespace)
 	}
 
-	if err = sessions.TrackVersion(ctx, tenant); err != nil {
-		return Response{}, err
+	if !option.DoNotReloadMetadata {
+		if err = sessions.TrackVersion(ctx, tenant); err != nil {
```

Comment:
Should we reload by default and put skip behind flag?

Reply:
Fixed.